initialization/admin.py

```diff
@@ -16,22 +23,32 @@ class AppView(ModelView):
     can_export = True
     can_view_details = True
 
+    def _prettify_name(self, name):
+        return re.sub(r'([a-z])([A-Z])', r'\1 \2', name).title()
```

Comment:
This special method is overwritten to format the column names in active admin and turn `camelCased` into `Camel Cased`.